client/state/plans/test/fixture.js

```diff
@@ -254,7 +252,7 @@ export const PLAN_2002 = {
 };
 
 export const WPCOM_RESPONSE = [
-	PLAN_1, PLAN_1003, PLAN_1008, PLAN_1009, PLAN_2000, PLAN_2001, PLAN_2002
+	PLAN_1, PLAN_1003, PLAN_1008, PLAN_2000, PLAN_2001, PLAN_2002
```

Comment:
it should. I've added the personal plan to the test fixture so `nock` returns all the plans.